webapp/shop/cred/views.py

```diff
@@ -573,6 +573,15 @@ def cred_shop_thank_you(**kwargs):
     )
 
 
+@shop_decorator(area="cube", permission="user", response="html")
+@canonical_staff()
+def cred_shop_keys(**kwargs):
+    cue_key_product = get_cue_products(type="keys").json
```

Comment:
Seems like in the JS code we were finding the product with `product.id == "cue-activation-key"`, we should probably do something similar here.